src/content/additional-search-engines.js

```diff
@@ -3,35 +3,21 @@
 (async function () {
 
   function cleanup() {
-    const $content = document.querySelector('.other-search-engines');
-    const $searchEngines = document.querySelector('.searchengines');
-    if ($searchEngines) {
-      $content.removeChild($searchEngines);
-    }
+    const $searchEngines = document.querySelector('.search-engines');
+    $searchEngines.innerHTML = '';
```

Comment:
I think MDN review will flag use of `innerHTML`. While this use is innocuous, we should try to avoid using it.